docs/sphinx/index.rst

```diff
@@ -11,14 +11,14 @@ You are browsing the documentation for |version| version of CUDA Quantum. You ca
    :caption: Contents
    :maxdepth: 2
 
-      Getting Started <install.rst>
-      Learn the Basics <using/cudaq.rst>
-      Advanced Topics  <using/advanced.rst>
-      Examples <using/examples.rst>
-      Tutorials <using/tutorials.rst>
-      Simulator Backends <using/simulators.rst>
-      Hardware Backends <using/hardware.rst>
+      Quick Start <quick_start.rst>
+      Basics <using/basics.rst>
+      Examples <using/examples/examples.rst>
+      Applications <using/tutorials.rst>
+      Backends <using/backends/backends.rst>
+      Installation <using/install/install.rst>
+      Integration <using/integration/integration.rst>
+      Extending <using/extending/extending.rst>
       Specifications <specification/index.rst>
       API Reference <api/api.rst>
-      Data Center Installation Guide <data_center_install.rst>
       Other Versions <versions.rst>
```

Comment:
I see that the `Other Versions` link was not changed, however, using the latest docs from CI -https://htmlpreview.github.io/?https%3A%2F%2Fgithub.com%2FNVIDIA%2Fcuda-quantum%2Fblob%2Fdocs%2Fpreview%2Fpr-1225%2Findex.html= , the other versions link isn't rendering, it shows this - https://[raw.githubusercontent.com/NVIDIA/cuda-quantum/docs/preview/latest/releases.html](https://raw.githubusercontent.com/NVIDIA/cuda-quantum/docs/preview/latest/releases.html)